server/state/state_store.go

```diff
@@ -0,0 +1,29 @@
+package state
+
+import (
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/server/sandbox"
+)
+
+// Store stores the state of the CRI-O server, including active pods and containers
+type Store interface {
```

Comment:
Sure. `golint` actually complained when it was named `StateStore` because writing `state.StateStore` is apparently overly redundant, so we'll have to remove that warning. But I agree that just `Store` is not a good name